src/TextTemplateService.Data.Provider.MsSql.Ef/Migrations/20220707231500_AddTemplateTexts.cs

```diff
@@ -157,6 +159,30 @@ protected override void Up(MigrationBuilder migrationBuilder)
             + "Перейдите по этой ссылке: https://dev.ltdo.xyz/auth/reactivate?userId={[Id]}\nПароль: {[Password]}",
             ru,
             true
+          },
+          {
+            Guid.NewGuid(),
+            SmtpCheckTemplate,
+            "SMTP settings check",
+            "SMTP settings check",
+            "Hello!\n"
+            + "This email address was specified as the administrator's email address to receive the SMTP check's message.\n"
+            + "If you haven't submitted a request, ignore this message, please.\n"
```

Comment:
maybe "if you haven't submitted a request, please ignore this message."?

Reply:
OK)